test/test_helper.rb

```diff
@@ -33,5 +29,5 @@ def login
     sign_in(user)
   end
 
-  EmailDomainValidator.stubs(:allowed_email_domains).returns(["alloweddomain.com"])
+  # EmailDomainValidator.stubs(:allowed_email_domains).returns(["alloweddomain.com"])
```

Comment:
We can delete that line completely.

Reply:
Resolved in https://github.com/appfolio/excelsior/pull/8/commits/f9e8fe3390d90305ab7e5197d89445fa80654558